<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmPickDialog</class>
 <widget class="QWidget" name="ConfirmPickDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>152</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>152</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>112</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(243, 243, 243);border:0.5px groove gray;border-radius:7px;padding:2px 4px;border-style: outset;}
QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
QPushButton:pressed{background-color:rgb(191, 190, 192);border-style: inset;}
QPushButton:disabled{background-color:rgb(191, 190, 192);border-style: inset;}</string>
   </property>
   <property name="text">
    <string>确认抽取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>112</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(243, 243, 243);border:0.5px groove gray;border-radius:7px;padding:2px 4px;border-style: outset;}
QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
QPushButton:pressed{background-color:rgb(191, 190, 192);border-style: inset;}
QPushButton:disabled{background-color:rgb(191, 190, 192);border-style: inset;}</string>
   </property>
   <property name="text">
    <string>取消抽取</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
